README.md

```diff
@@ -1044,6 +1044,7 @@ A GitHub conflict is when people make changes to the same area or line in a file
   - [Kajal Jaiswal](https://github.com/kajal1801)
   - [Kaleb P](https://github.com/Devarite)
   - [Kam Cherhesa](https://github.com/KazChe)
+  - [Kamalika Ghora] (https://github.com/doesnots)
```

Comment:
There's an extra space here.
```suggestion
  - [Kamalika Ghora](https://github.com/doesnots)
```

Reply:
Ohh Thank You for suggesting the changes. I'll do it right away!